homeassistant/components/zha/core/channels/base.py

```diff
@@ -43,6 +44,13 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+class ReportConfig(TypedDict, total=True):
+    """Matcher for the dhcp integration for required fields."""
+
+    attr: str
+    config: tuple(int, int, int)
```

Comment:
I couldn't find any instance where the third value can be a float. It seems to always be an int, but I can change it back if needed.